azkaban_cli/api.py

```diff
@@ -499,3 +499,30 @@ def fetch_flow_execution_request(session, host, session_id, exec_id):
     logging.debug("Response: \n%s", response.text)
 
     return response
+
+def fetch_flow_execution_updates_request(session, host, session_id, exec_id, last_update_time):
+    """Fetch a flow execution updates request for the Azkaban API
+
+    :param session: A session for creating the request
+    :type session: requests.Session
+    :param str host: Hostname where the request should go
+    :param str session_id: An id that the user should have when is logged in
+    :param str exec_id: Execution id to be fetched
```

Comment:
Missing docstring entry for `last_update_time`

Reply:
Added to **azkaban_cli/api.py** 
````
    :param last_update_time: The criteria to filter by last update time. Set the 
     value to be -1 if all job information are needed. Use -lt="value" to
     subscribe the default value, defaults to -1
    :type last_update_time: str, optional
````